auth_proxy/services/oauth.py

```diff
@@ -199,3 +201,35 @@ def cb_tokensetter(self, token, request, *args, **kwargs):
         self.db.session.commit()
 
         return new
+
+    def create_debug_token(self, client_id, approval_expires, scopes, user, patient_id):
+
+        creating_user = User.query.filter_by(username=user).one()
+        client = Client.query.filter_by(client_id=client_id).one()
+
+        token = Token(
+            client_id=client_id,
+            client=client,
+            approval_expires=arrow.get(approval_expires).datetime,
+            _scopes=scopes,
+            user=creating_user,
+            patient_id=patient_id
+        )
+
+        self.db.session.add(token)
+        self.db.session.commit()
+
+        generated_access_token = str(uuid.uuid4())
+        generated_refresh_token = str(uuid.uuid4())
+
+        new = token.refresh(generated_access_token,
```

Comment:
Can you add a quick comment documenting why refresh step is needed?

Reply:
Done! It's mostly for future proofing if we implement logic in the refresh that we'd want to be applied to 'debug' tokens too.